tests/test_citrine.py

```diff
@@ -23,15 +23,13 @@ def refresh_token(expiration: datetime = None) -> dict:
 def test_citrine_creation():
     with requests_mock.Mocker() as m:
         m.post('https://citrine-testing.fake/api/v1/tokens/refresh', json=token_refresh_response)
-        m.get('https://citrine-testing.fake/api/v1/utils/runtime-config', json=dict())
```

Comment:
Why is this line removed?

Reply:
That's the endpoint which was hit to retrieve the accounts version. Since the session no longer hits it, its presence on the mock was causing the test to fail.